nzedb/Menu.php

```diff
@@ -11,30 +11,40 @@ class Menu
 	public $pdo;
 
 	/**
-	 * @param \nzedb\db\DB $pdo
+	 * Menu constructor.
+	 *
+	 * @param null $pdo
```

Comment:
This is wrong. $pdo is of type \nzedb\db\DB as the original @param said. It cannot be null after the constructor